apps/sms/test/marionette/lib/views/new-message/accessors.js

```diff
@@ -14,6 +14,9 @@ var SELECTORS = Object.freeze({
   subjectInput: '.subject-composer-input',
   sendButton: '#messages-send-button',
   attachButton: '#messages-attach-button',
+  headerActionButton: '#messages-options-button',
```

Comment:
Well, here you should have selector for the action button (back button) that is placed inside header. Selector you use is for Options button. I suspect it should be ```.action-button```, but you need to check with Firefox or Simulator for example. Have you tried?